INFO     backend.layers.business.business:business.py:502 {'message': 'ingesting dataset', 'collection_version_id': e12451ae-1528-4e43-9256-538ebce063ca, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:502 {'message': 'ingesting dataset', 'collection_version_id': e12451ae-1528-4e43-9256-538ebce063ca, 'url': None, 'current_dataset_version_id': 199fd067-2ddc-47d5-8619-a66a61a84357}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [7b89ac70-70a9-4397-bbfd-116e5c0d4d9e/raw.h5ad] to s3://artifact_bucket/7b89ac70-70a9-4397-bbfd-116e5c0d4d9e/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:244 Main: Starting thread for h5ad update
INFO     processing:dataset_metadata_update.py:264 Main: Starting thread for rds update
ERROR    processing:dataset_metadata_update.py:302 Cannot find cxg artifact uri for 199fd067-2ddc-47d5-8619-a66a61a84357.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status